.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: bash
+sudo: required
+dist: trusty
+
+before_install:
+  - sudo apt-get update -q
+  - sudo apt-get install cabal-install -y
+  - cabal update
+  - cabal install shellcheck
```

Comment:
I'd prefer to install a specific version (say, 0.4.3) so that new releases of shellcheck don't cause an unexpected failure in Travis.

    - cabal install shellcheck-0.4.3